.gitignore

```diff
@@ -267,3 +267,6 @@ modules.xml
 .idea/sonarlint
 
 # End of https://www.gitignore.io/api/node,terraform,visualstudiocode,webstorm+all,macos,linux
+reports
+.gauge
+gauge-reports
```

Comment:
I don't like to idea to manually edit this generated .gitignore file. 
any better alternative?

Reply:
it should be modified from gitignore.io, but Gauge is not among the available options.
Extra exclusions can be put on top.

Reply:
but what is reports folder?